modules/gsp-cluster/data/values.yaml

```diff
@@ -30,6 +30,16 @@ notary:
   delegationPassphrase: ${notary_delegation_passphrase}
   delegationKey: ${notary_delegation_key}
 
+cluster-autoscaler:
+  cloudProvider: aws
+  awsRegion: eu-west-2
+  autoscalingGroups:
+    - name: ${cluster_name}-worker-nodes
+      minSize: ${min_workers}
+      maxSize: ${max_workers}
+  podAnnotations:
+    iam.amazonaws.com/role: ${cluster_autoscaler_role_name}
```

Comment:
This plumbing is unnecessary as we're targeting the kiam-server node group, so kiam won't apply (I think)

Reply:
I only put it on the kiam-server (aka cluster-management) nodegroup after discussion with @chrisfarms.

It's possible to put the cluster autoscaler on the worker nodegroup but you have to be super careful that the autoscaler doesn't delete the node that it's running on.  [The docs say](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#deployment):

> Cluster Autoscaler is designed to run on Kubernetes master node. This is the default deployment strategy on GCP. It is possible to run a customized deployment of Cluster Autoscaler on worker nodes, but extra care needs to be taken to ensure that Cluster Autoscaler remains up and running. Users can put it into kube-system namespace (Cluster Autoscaler doesn't scale down node with non-mirrored kube-system pods running on them) and set a priorityClassName: system-cluster-critical property on your pod spec (to prevent your pod from being evicted).

Reply:
I backed out the commit that put it on the kiam nodes, because those nodes should be going away soon.